linux_os/guide/services/ssh/ssh_server/sshd_rekey_limit/rule.yml

```diff
@@ -7,7 +7,7 @@ description: |-
     the session key of the is renegotiated, both in terms of
     amount of data that may be transmitted and the time
     elapsed. To decrease the default limits, put line
-    <tt>RekeyLimit 512M 1h</tt> to file <tt>/etc/ssh/sshd_config</tt>.
+    <tt>RekeyLimit {{{ sub_var_value("rekey_limit_size") }}} {{{ sub_var_value("rekey_limit_time") }}}</tt> to file <tt>/etc/ssh/sshd_config</tt>.
```

Comment:
Please keep the variable convention with these variables so that is easy to understand that these are variables

`s/rekey_limit_size/var_rekey_limit_size/g`
`s/rekey_limit_time/var_rekey_limit_time/g`

Reply:
Looks good. Thank you